pkg/collector/corechecks/oracle-dbm/activity_queries.go

```diff
@@ -150,7 +150,7 @@ WHERE
 	)
 	AND status = 'ACTIVE'`
 
-const activityQueryDirect = `SELECT /* DD_ACTIVITY_SAMPLING */
+const activityQueryDirect = `SELECT /*+ push_pred(sq) push_pred(sq_prev) */ /* DD_ACTIVITY_SAMPLING */
```

Comment:
It would require an article, as it's dealing with undocumented Oracle internals. Related to this: https://dd.slack.com/archives/C04HV5BPBU2/p1700668875163629 .